.travis.yml

```diff
@@ -1,14 +1,37 @@
+---
+os: osx
 sudo: true
-git:
-  depth: 10
+# lie to make travis work with nvm on osx
+language: cpp
+env:
+  matrix:
+    - TRAVIS_NODE_VERSION="0.10"
+    - TRAVIS_NODE_VERSION="0.12"
+    - TRAVIS_NODE_VERSION="2"
+    - TRAVIS_NODE_VERSION="4"
+    - TRAVIS_NODE_VERSION="5"
```

Comment:
You don't really have to run it with these node versions, Electron always uses it's bundled node core as the node runtime. You should be using different versions of Atom maybe if you want to ensure compatibility with older runtimes